package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "waasm-wordpress",
+  "name": "wasm-wordpress",
```

Comment:
Fixing a little typo - this change is unrelated to the main topic of the pull request, but I figured it's OK to include.

Reply:
Good call!